iidm/iidm-modification/src/main/java/com/powsybl/iidm/modification/topology/ModificationReports.java

```diff
@@ -0,0 +1,101 @@
+/**
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.iidm.modification.topology;
+
+import com.powsybl.commons.reporter.Report;
+import com.powsybl.commons.reporter.Reporter;
+import com.powsybl.commons.reporter.TypedValue;
+import com.powsybl.iidm.network.*;
+
+/**
+ * @author Coline Piloquet <coline.piloquet at rte-france.com>
+ */
+final class ModificationReports {
+    static String voltageLevelIdString = "voltageLevelId";
+
+    public static void notFoundBusbarSectionReport(Reporter reporter, String bbsId) {
```

Comment:
Should these methods be public or is package-private enough?

Reply:
At least for now, I guess package-private is enough